torch/_inductor/virtualized.py

```diff
@@ -96,9 +97,35 @@ def inner(*args):
 class KernelFormatterHandler:
     def __init__(self, parent_handler):
         self.parent_handler = parent_handler
-        self.output = IndentedBuffer()
+        self.output = IndentedBuffer(1)
         self.var_counter = itertools.count()
 
+    @staticmethod
+    def ir_to_string(ir_fn, index, rindex=None):
+        from .ir import FlexibleLayout
+
+        args = [index, rindex] if rindex is not None else [index]
+        names = ["index", "rindex"] if rindex is not None else ["index"]
+        formatter = KernelFormatterHandler(MockHandler())
+
+        with formatter.output.indent(-1):
+            formatter.output.writeline(f"def inner_fn({', '.join(names)}):")
+        for name, arg in zip(names, args):
+            if arg:
+                lhs = ", ".join(
+                    [
```

Comment:
This can be a generator instead of a list comprehension

Reply:
I don't think it matters since this list will be <10 elements.